content/07.supp_info.md

```diff
@@ -2,7 +2,7 @@
 
 A version of the main paper figures using the area under the receiver-operator curve (AUROC) metric rather than AUPR is available at [`https://doi.org/10.6084/m9.figshare.14919729`](https://doi.org/10.6084/m9.figshare.14919729).
 
-In a previous version of this paper, we ran our analysis only for the genes in the Vogelstein et al. [@doi:10.1126/science.1235122] gene set. Scaling up the gene set, by combining cancer gene sets from the literature as described in the methods/results sections, affected the study results somewhat, as mutations in the added genes tend to be better predicted using gene expression than other data types. During the revision, we explored the difference between the genes in this gene set, and the genes in the "merged" gene set but not in the Vogelstein genes. GO analysis results for the Vogelstein genes are available at [`https://doi.org/10.6084/m9.figshare.19565890`](https://doi.org/10.6084/m9.figshare.19565890), and results for the non-Vogelstein genes are available at [`https://doi.org/10.6084/m9.figshare.19565887`](https://doi.org/10.6084/m9.figshare.19565887). In general, we noticed that the non-Vogelstein genes tend to be enriched for terms relating to transcription factors and transcriptional regulation.
+In a previous version of this paper, we ran our analysis only for the genes in the Vogelstein et al. [@doi:10.1126/science.1235122] gene set. Scaling up the gene set, by combining cancer gene sets from the literature as described in the methods/results sections, affected the study results somewhat, as mutations in the added genes tend to be better predicted using gene expression than other data types. During the revision, we explored the difference between the genes in this gene set, and the genes in the "merged" cancer-related gene set but not in the Vogelstein genes. GO analysis results for the Vogelstein genes are available at [`https://doi.org/10.6084/m9.figshare.19565890`](https://doi.org/10.6084/m9.figshare.19565890), and results for the non-Vogelstein genes are available at [`https://doi.org/10.6084/m9.figshare.19565887`](https://doi.org/10.6084/m9.figshare.19565887). In general, we noticed that the non-Vogelstein genes tend to be enriched for terms relating to transcription factors and transcriptional regulation.
```

Comment:
```suggestion
In a previous version of this paper, we ran our analysis only for the genes in the Vogelstein et al. [@doi:10.1126/science.1235122] gene set.
While there were some gene-to-gene differences in this set, we did not observe large differences between methylation and gene expression performances overall.
Scaling up the gene set by combining cancer gene sets from the literature as described in the methods/results sections affected the study results somewhat, as mutations in the added genes tend to be better predicted using gene expression than other data types.
During the revision, we explored the difference between the genes in this gene set and the genes in the "merged" cancer-related gene set but not in the Vogelstein genes.
GO analysis results for the Vogelstein genes are available at [`https://doi.org/10.6084/m9.figshare.19565890`](https://doi.org/10.6084/m9.figshare.19565890), and results for the non-Vogelstein genes are available at [`https://doi.org/10.6084/m9.figshare.19565887`](https://doi.org/10.6084/m9.figshare.19565887).
We noticed that the non-Vogelstein genes tend to be enriched for terms relating to transcription factors and transcriptional regulation.
```

Reply:
Added one sentence in the middle there - feel free to undo this if the one sentence per line breaks things.